### ISSUE TITLE
[Feature] [DevTools]

### ISSUE THREAD

**Issue by souporserious** [motiondivision/motionone#122]
**Is your feature request related to a problem? Please describe.**
Using numbers for width/height animation does not currently work.

**Describe the solution you'd like**
Using numbers should work.

**Describe alternatives you've considered**
I'm currently parsing these myself by checking if the incoming value is a number and appending `px` if so.

**Additional context**
Totally understand if you don't want to support this since it's nonstandard, but found myself doing this a lot coming from React and wondering why things weren't working. If anything, maybe an error to use px would be nice.

**Comment by mattgperry** [comment]
Yeah I completely agree, I’ve run into this myself - and for the same reason
